site/src/comparison.rs

```diff
@@ -577,54 +604,39 @@ pub fn write_summary_table(
         },
     ]);
 
-    let largest_change = if primary.is_empty() {
-        "-".to_string()
-    } else {
-        let largest_improvement = primary
-            .largest_improvement()
-            .map(|c| c.relative_change())
-            .unwrap_or(0.0);
-        let largest_regression = primary
-            .largest_regression()
-            .map(|c| c.relative_change())
-            .unwrap_or(0.0);
-        let change = if largest_improvement
-            .abs()
-            .partial_cmp(&largest_regression.abs())
-            .unwrap_or(Ordering::Equal)
-            != Ordering::Less
-        {
-            largest_improvement
-        } else {
-            largest_regression
-        };
-
-        format!("{:.1}%", change * 100.0)
-    };
-
-    // max
+    // range
+    let f = |r: Option<&TestResultComparison>| r.unwrap().relative_change() * 100.0;
```

Comment:
I know we want to keep the name relatively terse, but the name `f` is very opaque...